use-timescale/compression/about-compression.md

```diff
@@ -154,13 +154,13 @@ showing numbers around 88, and device 2 showing numbers around 300:
 |-|-|-|-|-|
 |[12:00:02, 12:00:02, 12:00:01, 12:00:01 ]|[1, 2, 1, 2]|[88.2, 300.5, 88.6, 299.1]|[20, 30, 25, 40]|[0.8, 0.9, 0.85, 0.95]|
 
-To improve the performance of this data, you can order by `device_id, time DESC`
+To improve the performance of this data, you can order by `device_id, time ASC`
```

Comment:
```suggestion
To improve the performance of this data, you can order by `device_id, time`
```

Reply:
I do not think that `ASC` is the default (yet?) and the text should also provide good guidance for users before timescale/timescaledb#6299?